Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -361,7 +361,7 @@
 "wikitext-upload-save-anonymously-warning" = "Edits will be attributed to the IP address of your device. If you $1 you will have more privacy.";
 "wikitext-upload-save-license-name" = "CC BY-SA 3.0";
 "wikitext-upload-save-sign-in" = "Log in";
-"wikitext-upload-save-terms-and-license" = "By publishing, you agree to the $1, and to irrevocably release your contributions under the $2 license.";
+"wikitext-upload-save-terms-and-license" = "By saving changes, you agree to the $1 and agree to release your contribution under the $2 and $3 licenses.";
```

Comment:
We would need to change the key `wikitext-upload-save-terms-and-license` since`$3` is  being added. Perhaps to `wikitext-upload-save-terms-and-licenses`